information-schema/information-schema-collation-character-set-applicability.md

```diff
@@ -38,3 +38,13 @@ SELECT * FROM collation_character_set_applicability WHERE character_set_name='ut
 +----------------+--------------------+
 1 row in set (0.00 sec)
 ```
+
++----------------+--------------------+
+| COLLATION_NAME | CHARACTER_SET_NAME |
+| utf8mb4_bin | utf8mb4 |
++----------------+--------------------+
```

Comment:
lines 42 to 45 are duplicate with lines 34 to 38. Removed duplicates in https://github.com/pingcap/docs/pull/3470/commits/520218547d2599a300960c01109c676a02f4c640. (also for [the Chinese version](https://github.com/pingcap/docs-cn/pull/4109/files#diff-6d4157f625dd3f18efcbc5add886ad6cR34-R39))